kernel/ksu.c

```diff
@@ -244,6 +244,17 @@ static int handler_pre(struct kprobe *p, struct pt_regs *regs)
 		return 0;
 	}
 
+	// Both root manager and root processes should be allowed to get version
+	if (arg2 == CMD_GET_VERSION) {
+		if( is_manager() || 0 == current_uid().val ) {
+			u32 version = KERNEL_SU_VERSION;
+				if (copy_to_user(arg3, &version, sizeof(version))) {
```

Comment:
Incorrect indent

Reply:
I don't want CI to rerun, could you please fix it after merge?

Reply:
It is ok to let it run

Reply:
add `[skip ci]` to your commit message
https://docs.github.com/zh/actions/managing-workflow-runs/skipping-workflow-runs